doc/pages/Manual.md

```diff
@@ -1077,6 +1077,15 @@ files.
 - <a id="opamfield-build-env">`build-env: [ <environment-update> ... ]`</a>:
   defines environment updates that will be applied when running the package's
   build, install and remove scripts.
+  
+  By default, the following environment variables are set by opam:
+  + `CDPATH=`
+  + `MAKEFLAGS=`
+  + `MAKELEVEL=`
+  + `OPAM_PACKAGE_NAME=<pkg>` (with `<pkg>` being the name of the current package being built/installed/removed)
+  + `OPAM_PACKAGE_VERSION=<ver>` (with `<ver>` being the version of the current package being built/installed/removed)
+  + `OPAMCLI=2.0` (since opam 2.1)
+  + `TMP` and `TMPDIR` are also set by the sandbox (bubblewrap) script but are not to be relied upon.
```

Comment:
I'd reword this slightly to explain why:
```suggestion
  + `TMP` and `TMPDIR` are set by the sandbox script (bubblewrap), but should not be relied on since ...
```

Reply:
```suggestion
  + `TMP` and `TMPDIR` are set by the sandbox script (bubblewrap), but should not be relied on since the sandbox is not used on every platforms and can be disabled by the user.
```
What about this?

Reply:
LGTM!